app/cafe.py

```diff
@@ -0,0 +1,22 @@
+from datetime import date
+from app.errors import NotWearingMaskError, \
+    OutdatedVaccineError, \
+    NotVaccinatedError
+
+
+class Cafe:
+    def __init__(self, name: str):
+        self.name = name
+
+    def visit_cafe(self, visitor: dict):
+
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError

```

Comment:
you should pass descriptive messages when you raise the exception in the visit_cafe method